pkg/asset/machines/master.go

```diff
@@ -243,6 +267,40 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 	}
 
+	if len(hostSettings.Hosts) > 0 {
```

Comment:
why is this not in bare-metal switch??

Reply:
All of the other code to populate the list of files was outside of the switch, so I structured this new code the same way.

Reply:
but those are valid for all platforms, these aren't...

Reply:
OK, sure. It looked like the switch statement was for initializing the inputs to build the files, and then the file building happened later. I'll move all of the baremetal file stuff up inside the switch statement.